src/components/drawer/drawer.spec.js

```diff
@@ -14,6 +14,9 @@ import {
 jest.mock('../../utils/helpers/guid');
 guid.mockImplementation(() => 'guid-123');
 
+const container = { current: null };
+const wrapper = { current: null };
```

Comment:
could you explain what do we need these objects for?

Reply:
After chatting to Kuba and Jamie we agreed that this approach is an overkill. Simplified the specs. fixed.